src/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": null,
+    "database": "ecommerce_database",
+    "host": "127.0.0.1",
+    "dialect": "postgres"
+  },
+  "test": {
+    "username": "root",
+    "password": null,
+    "database": "database_test",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
+  },
+  "production": {
+    "username": "root",
+    "password": null,
+    "database": "database_production",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
+  }
+}
```

Comment:
For the person who worked on setting up the database and environments, they need to use the doting to manage these names.

So this does not concern you , and this is why you should bring in unfinished work aka unmerged changes.

Reply:
> For the person who worked on setting up the database and environments, they need to use the doting to manage these names.
> 
> So this does not concern you , and this is why you should bring in unfinished work aka unmerged changes.

Thank you @mukunzidd, @mansurissa  for your review and comments, I would like notify you that I have updated and resolved all the issues that has been raised before.  

Kindly you can re-review and merge my ``pull request`` to the `develop` if no other problem. Thanks.! [🙏](https://emojipedia.org/folded-hands/)

<img width="1318" alt="image" src="https://user-images.githubusercontent.com/46238001/222099428-6e241f03-ae7f-44c1-bc52-74e3ea390925.png">